### PR TITLE
connect owned groups

### PR DESCRIPTION
**Pull request by ConConRob**
# Description

connected owned groups

## Change status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: app/src/actions/groupActions.js
Comment (kamry-bowman) [inline app/src/actions/groupActions.js:17]: should remove debuggers in commits

File: app/src/components/SideBar/GroupsOwned.jsx
Comment (kamry-bowman) [inline app/src/components/SideBar/GroupsOwned.jsx:26]: need keys in your maps

**Review by kamry-bowman — APPROVED** [review]
Looks good, I would recommend to remove the debuggers before merging but will leave it up to you.

File: app/src/actions/groupActions.js
Comment (guillearria) [reply app/src/actions/groupActions.js:17]: Debuggers removed in upcoming PR.

File: app/src/components/SideBar/GroupsOwned.jsx
Comment (guillearria) [reply app/src/components/SideBar/GroupsOwned.jsx:26]: Keys added in upcoming PR.

**Pull request merged**
